### PR TITLE
Add reference traces

### PR DESCRIPTION
**Pull request by leokondrashov**
## Summary

Add reference traces created by sampling the Azure Functions Production Trace by `sampler` part of InVitro.

### CODE REVIEW

File: docs/sampler.md [hunk 1]
```diff
@@ -115,6 +111,28 @@ optional arguments:
...
+`preprocessed.tar.gz` contains the preprocessed traces for the original Azure trace for day 1, 00:09:00-00:11:30 (150
+minutes total).
+
+`sampled.tar.gz` contains the sampled traces for preprocessed trace from `preprocessed.tar.gz`. Sample sizes are 50-3k
+functions with step 50 and 3k-24k with step 1k.
+
+The reference traces were obtained by running the following commands:
+
+```console
+python3 -m preprocess  -t data/azure/ -o data/reference/preprocessed_150 -s 00:00:09 -dur 150
```
Comment (cvetkovic) [inline docs/sampler.md:128]: Inconsistency with the line 119. Did you start preprocessing from the 9th hour, or the 9th second?

**Review by cvetkovic — CHANGES_REQUESTED** [review]
LGTM! Just address the comment I wrote, and you can notify Dmitrii the job is done.

File: docs/sampler.md (hunk 1 above)
Comment (leokondrashov) [reply docs/sampler.md]: Yes, I made a mistake there. But the 9am is written as 00:09:00 because the format is dd:hh:mm, not the usual hh:mm:ss.

**Comment by leokondrashov** [comment]
@cvetkovic Please merge if it is fine now.

**Review by cvetkovic — APPROVED** [review]

**Pull request merged**